src/test-jdk17/java/com/fasterxml/jackson/databind/records/EnumNumberFormatShape3580PojoTest.java

```diff
@@ -0,0 +1,99 @@
+package com.fasterxml.jackson.databind.records;
```

Comment:
Probably should be in regular (non-jdk17) test dir, so test runs for JDK 8 too.

Reply:
Moved!